Patch to set the correct paths for CUDA
Thekla Loizou July 2015
--- lib/gpu/Makefile.linux.orig	2014-10-17 14:34:06.953493000 +0300
+++ lib/gpu/Makefile.linux	2014-10-17 14:32:34.558367000 +0300
@@ -7,7 +7,7 @@
 
 EXTRAMAKE = Makefile.lammps.standard
 
-CUDA_HOME = /usr/local/cuda
+CUDA_HOME = $(EBROOTCUDA)
 NVCC = nvcc
 
 # Tesla CUDA
--- lib/gpu/Makefile.lammps.standard.orig	2014-10-17 14:32:54.414313000 +0300
+++ lib/gpu/Makefile.lammps.standard	2014-10-17 14:33:12.184488000 +0300
@@ -2,4 +2,4 @@
 
 gpu_SYSINC =
 gpu_SYSLIB = -lcudart -lcuda
-gpu_SYSPATH = -L/usr/local/cuda/lib64
+gpu_SYSPATH = -L$(EBROOTCUDA)/lib64
